File: check_headers.py
import requests
import json

url = "https://mt-managwemnet.vercel.app/"
try:
    response = requests.get(url)
    print(f"Status Code: {response.status_code}")
    print("Headers:")
    print(json.dumps(dict(response.headers), indent=2))
except Exception as e:
    print(f"Error: {e}")

File: check_headers.js
import https from 'https';

const url = 'https://mt-managwemnet.vercel.app/';

https.get(url, (res) => {
  console.log('Status Code:', res.statusCode);
  console.log('Headers:', JSON.stringify(res.headers, null, 2));
}).on('error', (e) => {
  console.error('Error:', e);
});
